Tensile/Common.py

```diff
@@ -2617,3 +2626,15 @@ def __del__(self):
 """
 
 HR = "################################################################################"
+
+def supportedCompiler(compiler: str) -> bool:
+  """ Returns Determines if compiler is supported by Tensile
```

Comment:
Returns Determines?

The comment doesn't really match the rest of the code, are we moving towards this style?

The function name is a little misleading as it only checks if it's a Linux supported compiler, not Windows. Function should be moved to where the rest of the support functions are in this file and not after the header defines on it's own

Reply:
Should be `Determines` not `Returns Determines`. 

How do you feel about `supportedLinuxCompiler`? IIRC it is only used in cases where we are building binaries for Linux. Will update the location once we have agreement on the name.